test/pdo_sqlsrv/pdo_fetch_cursorBuffered_multicolumns.phpt

```diff
@@ -0,0 +1,271 @@
+--TEST--
+prepare with cursor buffered and fetch from numeric columns.
+--DESCRIPTION--
+Uses buffered cursor to fetch from float, int, and decimal columns that have positive, negative and zero value.
+--SKIPIF--
+
+--FILE--
+<?php
+
+require_once("autonomous_setup.php");
+$conn = new PDO( "sqlsrv:server=$serverName", $username, $password);
+$conn->setAttribute( PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION );
+$sample = 1234567890.1234;
+$sample1 = -1234567890.1234;
+$sample2 = 1;
+$sample3 = -1;
+$sample4 = 0.5;
+$sample5 = -0.55;
+
+$query = 'CREATE TABLE #TESTTABLE (a float(53), neg_a float(53), b int, neg_b int, c decimal(16, 6), neg_c decimal(16, 6), zero int, zerof float(53), zerod decimal(16,6))';
+$stmt = $conn->exec($query);
+$query = 'INSERT INTO #TESTTABLE VALUES(:p0, :p1, :p2, :p3, :p4, :p5, 0, 0, 0)';
+$stmt = $conn->prepare($query);
+$stmt->bindValue(':p0', $sample, PDO::PARAM_INT);
+$stmt->bindValue(':p1', $sample1, PDO::PARAM_INT);
+$stmt->bindValue(':p2', $sample2, PDO::PARAM_INT);
+$stmt->bindValue(':p3', $sample3, PDO::PARAM_INT);
+$stmt->bindValue(':p4', $sample4, PDO::PARAM_INT);
+$stmt->bindValue(':p5', $sample5, PDO::PARAM_INT);
+$stmt->execute();

```

Comment:
for bindValue() we don't have to supply the third parameter (it's actually confusing because most of these values are not integers anyway)